spec/controllers/exports_controller_spec.rb

```diff
@@ -0,0 +1,158 @@
+require 'rails_helper'
+
+RSpec.describe ExportsController, :type => :controller do
+  before do
+    FactoryGirl.create_list(:user, 3)
+    Delayed::Worker.delay_jobs = false  
+    ActionMailer::Base.delivery_method = :test
+  end
+
+  after do
+    Export.destroy_all
+  end
+
+  let(:valid_attributes) {
+    
+  }
+
+  let(:invalid_attributes) {
+    
+  }
+
+  # This should return the minimal set of values that should be in the session
+  # in order to pass any filters (e.g. authentication) defined in
+  # ExportsController. Be sure to keep this updated too.
+  let(:valid_session) { {} }
+
+  describe "GET index" do
+    it "assigns all exports as @exports" do
+      puts Export.count
+      export = Export.create! valid_attributes
```

Comment:
Please pull out all the useless files. The Rails generators can be helpful, but don't just commit all of them. Only keep what is being usied.